README.md

```diff
@@ -1,78 +1,96 @@
 
-# Terraform  module \<PROVIDER>-\<MODULE> Terraform module
-
-## How to use this template -- DELETE THIS SECTION BEFORE PR
-1. Create your new module repository by using terraform only (see SOP) and make sure to use this template.
-2. Add your Terraform code in any branch other than `main/master`
-3. Change the `<REPO_NAME>` value for any badges in the `README.md` files in the root `examples` and `modules` folders.
-4. Add nested modules in the `modules` folder, or `DELETE` the nested folder if not used.
-    * _Note: you will also maintain the nested modules full `README.md` files, remember nested modules can be used on their own._
-5. Add examples in the `examples` folder.
-    * _Note: you can have as many examples as you want, but two are required._
-        * _minimum - this is the example with the minimum code to use the module._
-        * _complete - this is the example with all features for a single module used (the most common use)._
-6. Run `pre-commit run --all-files` to update the `README` and fix any issues.
-    * _Note: make sure your IDE tool uses spaces and not tabs specially on `yaml` files._
-7. Run `checkov` or `terrascan` tool and make sure to add the log to the PR (something to automate).
-    * _Note: make sure to scan your module nested modules and examples (great candidate for a makefile action/script and automation)._
-8. Open a pull request into the default branch (usually `main`) and have it reviewed. don't forget to add the security scan logs.
-    * _Note: make sure to add the nested modules README's to the pre-commit config so they are also updated and validated._
-9. If you have been assigned a reviewer DM the reviewer, or the channel if it has been more than one day.
-10. Post to the channel news of the releases to the teams.
+# Terraform module AWS SCP Terraform module
 
 ## Description
 
-lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem.
+This module provides a resource to manage an AWS Organizations policy
 
-Examples available [`here`]github.com/boldlink/<REPO_NAME>//tree/main/examples)
+Examples available [`here`]github.com/boldlink/terraform-aws-scp/tree/main/examples)
 
 ## Usage
 *NOTE*: These examples use the latest version of this module
+- Run these examples on the `AWS MANAGEMENT ACCOUNT`.
 
 ```console
-module "miniumum" {
-  source  = "boldlink/<module_name>/<provider>"
-  version = "x.x.x"
-  <insert the minimum required variables here if any are required>
-  ...
+locals {
+  name = "minimum-scp-example"
+  content = jsonencode({
+    "Version" : "2012-10-17",
+    "Statement" : [
+      {
+        "Sid" : "RequireMicroInstanceType",
+        "Effect" : "Deny",
+        "Action" : "ec2:RunInstances",
+        "Resource" : [
+          "arn:aws:ec2:*:*:instance/*"
+        ],
+        "Condition" : {
+          "StringNotEquals" : {
+            "ec2:InstanceType" : "t2.micro"
+          }
+        }
+      }
+    ]
+    }
+  )
+}
+
+module "minimum_example" {
+  source  = "../../"
```

Comment:
The source on the readme should be using the actual terraform registry address `boldlink/scp/aws` and the examples the local folder ;)

To make it smaller don't specify the actual policy here, should be something like:

```console
module "minimum_example" {
  source  = "boldlink/scp/aws"
  name    = local.name
  content = local.content
}
```